Content/Items/Food/Sides.HoneySyrup.cs

```diff
@@ -22,7 +22,7 @@ public override void OnUseEffects(Player player, float multiplier)
 
 		public override void BuffEffects(Player Player, float multiplier)
 		{
-			Player.velocity.X *= 1 - 0.05f * multiplier;
+			Player.velocity.X *= 1 - 0.05f * multiplier;//todo: this isnt correct, and breaks running
```

Comment:
TODO?

Reply:
Not part of this pr, I just noticed a bug on a different item and wrote a comment on it